docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+
+x-uffizzi:
+  ingress:
+    service: pgadmin
+    port: 8081
+
+
+services:
+
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      - POSTGRES_USER=admin
+      - POSTGRES_PASSWORD=admin
+      - POSTGRES_DB=root
+      - PGDATA=/tmp
+    ports:
+      - "5432:5432"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    deploy:
+      resources:
+        limits:
+          memory: 250M
+
+  pgadmin:
+   image: "${PGADMIN_IMAGE}"
+   deploy:
+      resources:
+        limits:
+          memory: 250M
+   restart: always
+   environment:
+     - PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION=False
+     - PGADMIN_DEFAULT_EMAIL=admin@admin.com
+     - PGADMIN_DEFAULT_PASSWORD=secret
+     - PGADMIN_LISTEN_PORT=8081
+   ports:
+     - "5050:80"
+   depends_on:
```

Comment:
I think Uffizzi ignores the `restart`, `ports`, and `depends_on` stanzas?

Reply:
seems like depends_on and ports are definitely ignored. I do see restart in the manifest, maybe restart: Always is just the default.